src/elements/subsquid/Subsquid.wc.svelte

```diff
@@ -20,13 +20,25 @@
   import hasEnoughBalance from "../../utils/hasEnoughBalance";
   import validateName, { isInvalid ,validateEndpoint} from "../../utils/validateName"; // prettier-ignore
   import { noActiveProfile } from "../../utils/message";
+  import SelectCapacity from "../../components/SelectCapacity.svelte";
 
   let data = new Subsquid();
   let profile: IProfile;
 
   let loading = false;
   let success = false;
   let failed = false;
+  let diskField: IFormField;
+  let cpuField: IFormField;
+  let memoryField: IFormField;
+  data.disks = [new Disk()];
+
+  // define this solution packages
+  const packages: IPackage[] = [
+    { name: "Minimum", cpu: 2, memory: 1024 * 16, diskSize: 250 },
+    { name: "Standard", cpu: 2, memory: 1024 * 16, diskSize: 500 },
+    { name: "Recommended", cpu: 4, memory: 1024 * 16, diskSize: 1000 },
+  ];
```

Comment:
these resource requirements are high I think, can we re-check them?

Reply:
I lowered the requirements.